### PR TITLE
refactor: remove old plugfest test suite

### PR DESCRIPTION
**Pull request by tplooker**
Removes the old plugfest based test suite.

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

**Review by peacekeeper — CHANGES_REQUESTED** [review]
Why? We can add a comment that it has been superceded by the new one and shouldn't be used anymore, but I think it's worth preserving it in the repo so we have a record of it.

**Comment by tplooker** [comment]
@peacekeeper it will be preserved in history I just personally find it confusing to keep around a deprecated test suite in the repo.

**Review by fqutishat — APPROVED** [review]

**Comment by OR13** [comment]
@tplooker can you add a link to the old plugfest to the readme, so its not confusing.

**Comment by OR13** [comment]
IMO this is blocked by providing a link to the old suite in the readme... this kind of house cleaning is 100% necessary to ensure a positive first time contributor experience. its very bad form to keep dead / stale code around in the main branch.

**Comment by tplooker** [comment]
Ping @peacekeeper @msporny do you guys have any opinions on this?

**Comment by peacekeeper** [comment]
Okay let's include a link to the old suite in the README and then remove the old code, as suggested by @OR13 .

@tplooker do you want to add a commit to this PR to add such a link to README?

**Comment by tplooker** [comment]
@OR13 and @peacekeeper added a link to the old test suite.

**Review by peacekeeper — APPROVED** [review]

File: packages/vc-http-api-test-server/README.md [hunk 1]
```diff
@@ -33,4 +33,8 @@ npm run start-test-server
 curl -s -X POST http://localhost:8080/test-suite-manager/generate-report \
 -H "Content-Type: application/json" \
 -d @./suites/did-spec/default.json
-```
\ No newline at end of file
+```
+
+### Relationship to previous test suite
+
+This test suite supersedes the original which is available [here](https://github.com/w3c-ccg/vc-http-api/tree/b4df10dfdce98b453a2333e7ec1728a10bcc54d1/packages/plugfest-2020)
```
Comment (TallTed) [inline packages/vc-http-api-test-server/README.md:40]:
```suggestion
This test suite is meant to replace the original, which remains available 
[here](https://github.com/w3c-ccg/vc-http-api/tree/b4df10dfdce98b453a2333e7ec1728a10bcc54d1/packages/plugfest-2020) 
for reference purposes.
```

**Pull request merged**